lib/mongo/server/monitor/connection.rb

```diff
@@ -131,7 +133,7 @@ def initialize(address, options = {})
         #
         # @since 2.0.0
         def timeout
-          @timeout ||= options[:connect_timeout] || CONNECT_TIMEOUT
+          @timeout ||= options[:connect_timeout]
```

Comment:
Yes, I agree we should have one constant but I thought it was awkward/bad design for the address to reference the Server::Monitor::Connection's constants or vice-versa. I'll do some more thinking and see if I can do use a connect timeout default constant better.